extras/src/test/java/com/google/gson/typeadapters/RuntimeTypeAdapterFactoryTest.java

```diff
@@ -167,6 +167,25 @@ public void testSerializeCollidingTypeFieldName() {
     }
   }
 
+  public void testSerializeWrappedNullValue() {
+    TypeAdapterFactory billingAdapter = RuntimeTypeAdapterFactory.of(BillingInstrument.class)
```

Comment:
Which code from `internal` does my new test reference? The existing test code did import those packages but the test I have added does not (I believe).